composer.json

```diff
@@ -1,71 +1,75 @@
 {
-  "name": "spryker-eco/payone",
-  "type": "library",
-  "description": "Payone module",
-  "license": "MIT",
-  "require": {
-    "php": ">=7.3",
-    "spryker/calculation": "^4.0.0",
-    "spryker/cart": "^5.0.0 || ^7.0.0",
-    "spryker/config": "^3.0.0",
-    "spryker/customer": "^7.0.0",
-    "spryker/glossary": "^3.0.0",
-    "spryker/guzzle": "^2.1.0",
-    "spryker/kernel": "^3.33.0",
-    "spryker/oms": "^6.0.0 || ^7.0.0 || ^8.0.0 || ^10.0.0 || ^11.0.0",
-    "spryker/propel-orm": "^1.0.0",
-    "spryker/quote": "^2.14.0",
-    "spryker/refund": "^4.0.0 || ^5.0.0",
-    "spryker/sales": "^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0 || ^10.0.0 || ^11.0.0",
-    "spryker/shipment": "^6.0.0 || ^8.0.0",
-    "spryker/step-engine": "^3.0.0",
-    "spryker/symfony": "^3.0.0",
-    "spryker/util-encoding": "^2.0.0",
-    "spryker/zed-request": "^3.0.0"
-  },
-  "require-dev": {
-    "spryker/application": "^3.0.0",
-    "spryker/checkout": "*",
-    "spryker/code-sniffer": "*",
-    "spryker/country": "^3.2.0",
-    "spryker/payment": "^4.0.0 || ^5.0.0 ",
-    "spryker/propel": "*",
-    "spryker/router": "*",
-    "spryker/silex": "*",
-    "spryker/testify": "*"
-  },
-  "suggest": {
-    "spryker/checkout": "If you want to use Checkout plugins you need to install spryker/checkout.",
-    "spryker/application": "If you want to use Application plugins you need to install spryker/application.",
-    "spryker/payment": "If you want to use Payment plugins you need to install spryker/payment.",
-    "spryker/router": "Use this module if you want use the RouterProviderPlugin.",
-    "spryker/silex": "If you want to use Silex plugins you need to install spryker/silex."
-  },
-  "autoload": {
-    "psr-4": {
-      "SprykerEco\\": "src/SprykerEco/"
+    "name": "spryker-eco/payone",
+    "type": "library",
+    "description": "Payone module",
+    "license": "MIT",
+    "require": {
+        "php": ">=7.3",
+        "spryker/calculation": "^4.0.0",
+        "spryker/cart": "^5.0.0 || ^7.0.0",
+        "spryker/checkout-extension": "^1.3.0",
+        "spryker/config": "^3.0.0",
+        "spryker/customer": "^7.0.0",
+        "spryker/glossary": "^3.0.0",
+        "spryker/guzzle": "^2.1.0",
+        "spryker/kernel": "^3.33.0",
+        "spryker/oms": "^6.0.0 || ^7.0.0 || ^8.0.0 || ^10.0.0 || ^11.0.0",
```

Comment:
* Add the dependency ` "spryker/store": "^1.1.0".

Reply:
++